tpch/cudf_queries/queries.py

```diff
@@ -0,0 +1,1316 @@
+import cudf.pandas
+cudf.pandas.install()
+import pandas as pd
+
+import argparse
+import json
+import os
+import time
+import traceback
+from typing import Dict
+
+import sys
+# 将父目录添加到 Python 的搜索路径中
```

Comment:
Please do not use Chinese.
And keep consistent with other query files.

Reply:
The python compiler recommends this, and in the official documentation: Deprecated since version 2.2.0:" Setting include_groups to True is deprecated. Only the value False will be allowed in a future version of pandas."